playground/backend/internal/code_processing/code_processing_test.go

```diff
@@ -561,3 +568,66 @@ func Test_setJavaExecutableFile(t *testing.T) {
 		})
 	}
 }
+
+func Test_getRunOrTestCmd(t *testing.T) {
+	unitTests := make(map[string]bool, 1)
+	unitTests[validators.UnitTestValidatorName] = true
+
+	notUnitTests := make(map[string]bool, 1)
+	notUnitTests[validators.UnitTestValidatorName] = false
+
+	runEx := executors.NewExecutorBuilder().
+		WithRunner().
+		WithCommand("runCommand").
+		WithArgs([]string{"arg1"}).
+		Build()
+
+	testEx := executors.NewExecutorBuilder().
+		WithTestRunner().
+		WithCommand("testCommand").
+		WithArgs([]string{"arg1"}).
+		Build()
+
+	wantRunExec := exec.CommandContext(context.Background(), "runCommand", "arg1", "")
+	wantTestExec := exec.CommandContext(context.Background(), "testCommand", "arg1", "")
+
+	type args struct {
+		valResult      map[string]bool
+		executor       *executors.Executor
+		ctxWithTimeout context.Context
+	}
+
+	tests := []struct {
+		name string
+		args args
+		want *exec.Cmd
+	}{
+		{
+			//Get cmd objects with set run executor
+			name: "get run cmd",
+			args: args{
+				valResult:      notUnitTests,
+				executor:       &runEx,
+				ctxWithTimeout: context.Background(),
+			},
+			want: wantRunExec,
+		},
+		{
+			//Get cmd objects with set test executor
+			name: "get test cmd",
+			args: args{
+				valResult:      unitTests,
+				executor:       &testEx,
+				ctxWithTimeout: context.Background(),
+			},
+			want: wantTestExec,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := getExecuteCmd(tt.args.valResult, tt.args.executor, tt.args.ctxWithTimeout); !reflect.DeepEqual(got, tt.want) {
```

Comment:
Seems like we have an error here 
code_processing_test.go:628:30: cannot use tt.args.valResult (variable of type *map[string]bool) as map[string]bool value in argument to getRunOrTestCmd (compile)

Reply:
Everything is fine, can't see the problem